src/test/java/com/wikia/webdriver/common/dataprovider/VisualEditorDataProvider.java

```diff
@@ -92,6 +92,21 @@ public By getTag() {
     ADVANCED
   }
 
+  public enum Transclusion {
```

Comment:
I would move it to another class, data provider seems to be a bad place for such a nested class